### PR TITLE
Noticket - code cleanup

### PR DESCRIPTION
**Pull request by iozzyi**

### CODE REVIEW

File: frontend/src/components/Header/StopClock.js [hunk 1]
```diff
@@ -60,31 +59,31 @@ class StopClock extends Component<Props, State> {
...
-      this.props.history.push('/afterward');
+      this.props.onTimeout();
     }
   }
 
   componentWillUnmount() {
     clearInterval(this.timer);
   }
 
+  timer: IntervalID = setInterval(this.tick, 500);
```
Comment (jkimbo) [inline frontend/src/components/Header/StopClock.js]: Why did you move this out of `componentDidMount`? That's the best place for setting up timers.

Comment (iozzyi) [reply frontend/src/components/Header/StopClock.js]: @jkimbo It was kind of forced by Flow. IntervalID should only be an int or null (ever) but flow doesn't allow null or int, only the IntervalID type which is returned from a setInterval. So you have to initialise the field with a dummy setInterval i.e. `timer: IntervalID = setInterval(() => {}, 500)` ... but then in this case I could've just initialised with the real tick method
Happy to use the dummy interval or eslint-disable on this line if you think that's better

Comment (iozzyi) [reply frontend/src/components/Header/StopClock.js]: yeah that works :)
